src/pretix/static/pretixpresale/scss/_calendar.scss

```diff
@@ -2,58 +2,80 @@
   td, th {
     width: 14.29%;
   }
-  h3 {
-    margin: 0;
-    font-size: 16px;
+  p {
+    margin-bottom: 0;
+  }
+  .events {
+    list-style: none;
+    padding: 0;
   }
   a.event {
     display: block;
-    background: $brand-primary;
-    color: white;
-    border-radius: $border-radius-large;
+    background: lighten($brand-primary, 48%);
+    color: $brand-primary;
+    border-radius: $border-radius-base;
+    border-style: solid;
+    border-color: lighten($brand-primary, 30%);
+    border-width: 1px 1px 1px 12px;
+    border-left-color: inherit;
+
     padding: 3px 5px;
     margin-top: 3px;
     font-size: 12px;
     overflow-wrap: anywhere;
+    text-decoration: none;
 
-    &.continued {
-      background: #888;
-      opacity: 0.8;
-      &:hover {
-        background: darken(#888, 15%);
-        opacity: 1;
-      }
+    &:hover {
+      background: lighten($brand-primary, 50%);
+      border-color: $brand-primary;
+    }
+    &:focus {
+      outline-color: inherit;
     }
 
-    &.soon {
-      opacity: 0.8;
+    &.continued, &.over {
+      background: lighten(#767676, 48%);
+      border-color: lighten(#767676, 30%);
+      border-left-color: #767676;
```

Comment:
In a calender full of dozens of unavailable or sold out events, it's now much harder to spot the available ones, at least for me as someone with a very very small degree of red-green color blindness. The old layout was really easy for me to spot.

Maybe we should lighten the border color for the "over" and maybe even "soldout" states even more so that the green ones stick out?

![20210929_165855](https://user-images.githubusercontent.com/64280/135299032-a9c099aa-0260-41c8-bb24-68501aef077f.png)

Reply:
Better? Already pushed.

![Bildschirmfoto 2021-09-29 um 22 13 49](https://user-images.githubusercontent.com/276509/135341693-4d3a12cb-9dc8-4ed7-9ee6-1d6021ee5ff7.png)

Reply:
yes, much better!